src/CommunityToolkit.Maui/Behaviors/PlatformBehaviors/IconTintColor/IconTintColorBehavior.windows.cs

```diff
@@ -0,0 +1,234 @@
+﻿using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Numerics;
+using Microsoft.Maui.Platform;
+using Microsoft.UI.Composition;
+using Microsoft.UI.Xaml;
+using Microsoft.UI.Xaml.Hosting;
+using Microsoft.UI.Xaml.Media;
+using Microsoft.UI.Xaml.Media.Imaging;
+using WButton = Microsoft.UI.Xaml.Controls.Button;
+using WImage = Microsoft.UI.Xaml.Controls.Image;
+
+namespace CommunityToolkit.Maui.Behaviors;
+
+public partial class IconTintColorBehavior
+{
+	SpriteVisual? spriteVisual;
+	Vector2? originalImageSize;
+	bool IsUpdate => originalImageSize is not null;
```

Comment:
Let's make this variable name more descriptive. 
* Is something being updated or is something updated?
* What is being updated?

Reply:
Basically it was to differentiate in general between the initial apply of the behavior or a later update by a property change. I don't need to handle this case explicitly anymore since I use the `ImageSource.ImageOpened` to get notified when the image size is ready. So it doesn't matter if its the initial apply of the behavior or coming from a property change.